src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -5,4 +5,25 @@ class SimidMapCreative extends BaseSimidCreative {
   constructor() {
     super();
   }
+  /** @override */
+  onStart(eventData){
+    super.onStart(eventData);
+    this.adParams = JSON.parse(this.creativeData.adParameters);
+    console.log("Ad params: ", this.adParams);
+    var paramObject = this.adParams[0];
+    console.log(paramObject);
+    var buttonParam = paramObject["buttonParam"];
+    console.log(buttonParam);
+    this.specifyButtonText(buttonParam);
+  }
+ 
+  specifyButtonText(buttonParam) {
```

Comment:
This can be a private function.

annotate it with `/** @private */`, and end the name of it with an underscore `_`

Reply:
Added

Reply:
fixed